data/tabular/bicerano_dataset/meta.yaml

```diff
@@ -0,0 +1,95 @@
+---
+name: polymer_data
+description: |-
+    This paper outlines a MD simulation workflow based on GPU MD simulation and the
+    refined optimized potentials for liquid simulation (OPLS) OPLS3e force field to
+    calculate glass transition temperatures (Tgs) of 315 polymers for which Bicerano 
+    reported experimental values.
+targets:
+    - id: Tg_exp
+      description: experimental glass transition temperature
+      units: K
+      type: float
+      names:
+          - noun: experimental glass transition temperature 
+      uris:
+    - id: Tg_calc
+      description: calculated glass transition T
+      units: K
+      type: float
+      names:
+          - noun: computed glass transition temperature
+    - id: Tg_calc_std
+      description: computed standard deviation for glass transition temperature Tg
+      sampled: False
+      units: K
+      type: float
+      names:
+          - noun: computed standard deviation for glass transition temperature 
+    - id: rho_300K_exp
+      description: experiment density at 300K 
+      units: g/cm^3
+      type: float
+      names:
+          - noun: experimental density 300K
+    - id: rho_300K_calc
+      description: computed density at 300K 
+      units: g/cm^3
+      type: float
+      names:
+          - names: computed polymer density at 300K
+    - id: glass_CTE_calc
+      description: computed glass coef of thermal expansion (CTE) at 300K 
+      units: K
+      type: float
+      names:
+          - noun: computed glass coefficient of thermal expansion at 300K
+    - id: glass_CTE_calc_std
+      description: computed standard deviation of glass coefficient of thermal expansion (CTE) at 300K 
+      sampled: False
+      units: K
+      type: float
+      names:
+          - noun: computed standard deviation of glass CTE
+    - id: rubber_CTE_calc
+      description: computed rubber CTE at 300K 
+      units: K
+      type: float
+      names:
+          - noun: computed rubber coefficient of thermal expansion (CTE) at 300K 
+    - id: rubber_CTE_calc_std
+      description: computed standard deviation of rubber CTE at 300K 
+      sampled: False
+      units: K
+      type: float
+      names:
+          - computed standard deviation of rubber coefficient of thermal expansion (CTE) at 300K 
+identifiers:
+    - id: PSMILES
+      type: PSMILES

```

Comment:
this has to be changed in the data schema, too https://github.com/OpenBioML/chemnlp/blob/0710cf53cbf72dd92e97577d65b453cbb118b3e0/src/chemnlp/data_val/model.py#L9

Reply:
done